frontend/src/pages/ActivityReport/__tests__/index.js

```diff
@@ -85,6 +85,15 @@ describe('ActivityReport', () => {
     });
   });
 
+  it('hides program type if non-grantee is selected', async () => {
```

Comment:
Can we switch the description to match the behavior now - hidden until shown if grantee is selected.